Rakefile

```diff
@@ -6,3 +6,15 @@
 require_relative 'config/application'
 
 Rails.application.load_tasks
+
+begin
+  require 'rspec/core/rake_task'
+  require 'rubocop/rake_task'
+
+  desc 'Run linter and tests'
+  task ci: %i[spec rubocop]
```

Comment:
Whaddya think about running `rubocop` before `spec` so we fail fast when there are style-related issues?

Reply:
sure, i can change that, np.